cli/cmd/root.go

```diff
@@ -207,13 +292,17 @@ func initLogger() {
 func newCli() (*cli.CLI, error) {
 	var manager state.Manager
 	var presenters []presenter.Presenter
+	var updater cicdupdater.Updater
+
 	var err error
 
 	if config == nil {
 		return nil, errors.New("families config must not be nil")
 	}
 
-	if server != "" {
+	updater = cicdupdater.NewDefaultUpdater()
+
+	if (server != "" && !cicdMode) || (cicdMode && exportCICDResults) {
```

Comment:
what if `(cicdMode && exportCICDResults)` but server is empty?

Reply:
It can't be empty because of the validation of the flags:
	`validateRequiredFlagForDefinedFlag(rootCmd, "export-cicd-results", "server")`
	
If the export is set the server must be set as well.

Reply:
I would add a comment here, it just seems weird

Reply:
This piece of code was removed because of a small refactor. So it isn't relevant anymore.